doctr/__main__.py

```diff
@@ -63,9 +63,7 @@ def main():
 
 """.format(encrypted_variable=encrypted_variable.decode('utf-8'))
 
-        print("Put\n", travis_content, "in your .travis.yml.\n",
-            "Also make sure to create an empty gh-pages branch on GitHub, and "
-            "enable it at https://github.com/{repo}/settings".format(repo=repo), sep='')
```

Comment:
I just tested this -- I'm not sure when they changed it, but if you push up a `gh-pages` branch it is automatically enabled.